doc/source/tutorial/tutorial_aggregations.ipyml

```diff
@@ -86,6 +86,35 @@ cells:
     population.sum(gender, (benelux, fr_de))
 
 
+- markdown: |
+    <div class="alert alert-warning">
+    
+    **Warning:** Mixing slices and individual labels inside the `[ ]` will generate a **tuple of groups** instead of unique group.<br>If you want to create a unique group using both slices and individual labels, you need to use the `.union()` method (see below).   
```

Comment:
s/a tuple of/several distinct/ ?
s/of unique/of a single/
s/a unique/a single/